pkg/controller/registry/mem.go

```diff
@@ -220,19 +220,6 @@ func (m *InMem) setOrReplaceCRDDefinition(crd v1beta1.CustomResourceDefinition)
 func (m *InMem) findServiceConflicts(csv v1alpha1.ClusterServiceVersion) []error {
 	errs := []error{}
 
-	// validate required CRDs
```

Comment:
currently we're using this logic in `cmd/validator` to validate catalog resources. 

we don't need this now, but let's make an issue to remember to add options for validating that given a _set_ of catalogsources, all CRD requirements exist.

Reply:
Noted in ALM-646